apps/frontend/core-dapp/src/features/trade/TradeStore.ts

```diff
@@ -568,36 +569,45 @@ export class TradeStore {
   }
 
   /**
-   * The fee directed to prePO for this trade, in wstETH.
+   * The fees directed to prePO for this trade, in wstETH.
    *
    * If the user is paying with collateral (preWstEth), then this value is
    * exact. If the user is paying with native ETH, then this value is estimated,
    * since it depends on the Balancer quote.
    *
    * This does not include fees paid to Balancer or Uniswap.
    */
-  private get openTradeFeeInWstEthBN(): BigNumber | undefined {
-    const { openTradeAmountAfterDepositSlippageInWstEthBN, paymentToken } = this
+  private get openTradeFeesInWstEthBN(): FeeStore[] {
```

Comment:
technically the returned value isn't BigNumber nor in wst eth so this should be renamed to openTradeFees right?
```suggestion
  private get openTradeFees(): FeeStore[] {
```

Reply:
Good catch!